### PR TITLE
[VEN-2090]: multisig proposal for remote XVS bridge configurations in Sepolia

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

This PR is a proposal for configuring remote xvs bridge with certain limits and permissions

Resolves VEN-2090

### CODE REVIEW

File: multisig/proposals/vip-003/vip-003-sepolia-2.ts
Comment (chechu) [inline multisig/proposals/vip-003/vip-003-sepolia-2.ts:1]: Do we need to keep `vip-003-sepolia-2.ts`? Why don't we overwrite it?

File: multisig/simulations/vip-003/vip-003-sepolia-2/simulations.ts
Comment (chechu) [inline multisig/simulations/vip-003/vip-003-sepolia-2/simulations.ts:1]: Why not overwrite the first simulation?

File: multisig/proposals/vip-003/vip-003-sepolia-2.ts
Comment (GitGuru7) [reply multisig/proposals/vip-003/vip-003-sepolia-2.ts:1]: To make review easy, Anyway I overwrited here [d3e465e](https://github.com/VenusProtocol/vips/pull/141/commits/d3e465ef3519c8bc70fd92a086ece8a5e3d83ac1)

File: multisig/simulations/vip-003/vip-003-sepolia-2/simulations.ts
Comment (GitGuru7) [reply multisig/simulations/vip-003/vip-003-sepolia-2/simulations.ts:1]: [d3e465e](https://github.com/VenusProtocol/vips/pull/141/commits/d3e465ef3519c8bc70fd92a086ece8a5e3d83ac1)

**Comment by GitGuru7** [comment]
https://sepolia.etherscan.io/tx/0xfce76682204e698437f48e88584344c117eae9d459e5ffa45a98ebb40abfa33c

**Review by chechu — APPROVED** [review]

**Pull request merged**